src/main/java/homer/view/javafx/sensorsview/SensorDashboardViewManager.java

```diff
@@ -0,0 +1,137 @@
+package homer.view.javafx.sensorsview;
+
+import java.net.URL;
+import java.util.Objects;
+import java.util.ResourceBundle;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+
+import homer.controller.impl.electricalmeter.ElectricalMeterImpl;
+import homer.model.airquality.AirqualitySensor;
+import homer.model.environment.HomeEnvironment;
+import homer.model.thermometer.SimpleThermometer;
+
+/**
+ * Generates and updates the sensors dashboard.
+ */
+public final class SensorDashboardViewManager {
+    @FXML
+    // Reference of consumptionLabel for the FXML loader.
+    private Label consumptionLabel;
+
+    @FXML
+    // Reference of powerLabel for the FXML loader.
+    private Label powerLabel;
+
+    @FXML
+    // Reference of temperatureLabel for the FXML loader.
+    private Label temperatureLabel;
+
+    @FXML
+    // Reference of PM10Label for the FXML loader.
+    private Label pm10Label;
+
+    @FXML
+    // Reference of PM25Label for the FXML loader.
+    private Label pm25Label;
+
+    @FXML
+    // Reference of CO2Label for the FXML loader.
+    private Label co2Label;
+
+    @FXML
+    // Reference of ToxicGasLabel for the FXML loader.
+    private Label toxicGasLabel;
```

Comment:
aren't we supposed to store air quality info in the main view? Just asking, but it's perfectly fine if you need to. Just check out the changes introduced by #200 if you actually need the values.

Reply:
As the comment says, it's just a reference for the FXML loader.

Reply:
Oh, ok. I never really used FXML, so i thought this were like opt-in.